src/Voron/Data/BTrees/TreeIterator.cs

```diff
@@ -20,10 +21,11 @@ public unsafe class TreeIterator : IIterator
         private Slice _currentKey = default(Slice);
         private Slice _currentInternalKey = new Slice();
 
-        public TreeIterator(Tree tree, LowLevelTransaction tx)
+        public TreeIterator(Tree tree, LowLevelTransaction tx, bool prefetch)
         {
```

Comment:
You can use:
```public TreeIterator(Tree tree, LowLevelTransaction tx, bool prefetch = false )```
and that will requiere less changes to the code and also make it read better. Will only have a parameter there when we explicitely want to control the prefetching.